app/src/test/java/com/rickyhu/hushkeyboard/ui/HomeScreenUiTest.kt

```diff
@@ -17,10 +22,12 @@ class HomeScreenUiTest {
     @get:Rule
     val composeTestRule = createComposeRule()
 
+    var clicked = false
```

Comment:
Consider making this variable private and use a more descriptive name.

```suggestion
    private var isSettingsButtonClicked = false
```